app/src/main/java/org/mozilla/fenix/settings/SyncPreferenceView.kt

```diff
@@ -74,31 +73,33 @@ class SyncPreferenceView(
      */
```

Comment:
This kDoc needs to be updated still since we do not have the "On"/"Off labels anymore. I would revise this to say it shows a switch toggle for the sync preference when the user is logged in.